bin/webpack.js

```diff
@@ -333,16 +333,19 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 		var compiler;
 		try {
 			compiler = webpack(options);
-		} catch(e) {
-			var WebpackOptionsValidationError = require("../lib/WebpackOptionsValidationError");
-			if(e instanceof WebpackOptionsValidationError) {
+		} catch(err) {
+			if(err.name === "WebpackOptionsValidationError") {
```

Comment:
Could you write a test for this? A simple one in `test/binCases`.

Reply:
Yup, done